aries_vcx_core/Cargo.toml

```diff
@@ -12,6 +12,7 @@ modular_libs = ["dep:indy-vdr", "dep:indy-credx"]
 ########################## TEST FLAGS ###############################
 # Feature flag for base requirments for running tests underneath the `test_utils` flag (most tests).
 # Both `vdrtools` and `modular_libs` are transitively enabled, so they can be used for tests (including interop tests).
+mock = ["dep:mockall"]
```

Comment:
I'd suggest different feature name to better distinguish it / make it easier to look up. Also short description what code does the future enables would be nice

Reply:
Removed.